api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -831,11 +832,29 @@ def _update_venues_criteria(
 
 def _update_permanent_venues(venues: list[offerers_models.Venue], is_permanent: bool) -> list[offerers_models.Venue]:
     venues_to_update = [venue for venue in venues if venue.isPermanent != is_permanent]
+    actions = []
+
+    modified_info = {}
+    if is_permanent:
+        modified_info["permanent"] = {"old_info": False, "new_info": True}
+    if not is_permanent:
+        modified_info["permanent"] = {"old_info": True, "new_info": False}
+
     for venue in venues_to_update:
         venue.isPermanent = is_permanent
         if is_permanent and venue.thumbCount == 0:
             transactional_mails.send_permanent_venue_needs_picture(venue)
 
+        history = add_action(
+            action_type=history_models.ActionType.INFO_MODIFIED,
+            author=current_user,
+            venue=venue,
+            modified_info=modified_info,
+        )
+        actions.append(history)
+    if actions:
+        db.session.flush()
+
     return venues_to_update
```

Comment:
Plus clairement (à tester) : 
```suggestion
def _update_permanent_venues(venues: list[offerers_models.Venue], is_permanent: bool) -> list[offerers_models.Venue]:
    venues_to_update = [venue for venue in venues if venue.isPermanent != is_permanent]

    for venue in venues_to_update:
        history = add_action(
            action_type=history_models.ActionType.INFO_MODIFIED,
            author=current_user,
            venue=venue,
            modified_info={"isPermanent": {"old_info": venue.isPermanent, "new_info": is_permanent}},
        )
        venue.isPermanent = is_permanent
        if is_permanent and venue.thumbCount == 0:
            transactional_mails.send_permanent_venue_needs_picture(venue)

    db.session.flush()

    return venues_to_update
```